src/App.jsx

```diff
@@ -14,33 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const [sortType, setSortType] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const handleSort = type => {
+    const sortedGoods = [...goodsFromServer];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (type === 'alphabetically') {
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+    } else if (type === 'by length') {
+      sortedGoods.sort((a, b) => a.length - b.length);
+    }
```

Comment:
Use a switch case here and create a enum for 'by length', 'alphabetically' and use it everywhere